src/config.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Gregorio configuration headers. Copyright (C) 2015 Elie Roux
```

Comment:
Why not, but I'd prefer not to see my name at all... It would allow to have the same copyright notice everywhere (putting my name in files you create doesn't make much sense)